### DISCUSSION TITLE
Multiple instances on the same server - resources?

### DISCUSSION THREAD

**Discussion by abdessalaam** [mastodon/mastodon#22062] [Q&A]
Does anyone have any insight into this:
– how much of a server resources would you need to run two instances, both in docker containers?
Would 3 vCPU / 4GB RAM be fine with that? for small public instances initially, perhaps scaling up later.

Following up on a slightly related old [issue 7972](https://github.com/mastodon/mastodon/issues/7972)

**Comment by supernovae** [comment]
I wouldn't try and run multiple instances in 4GB of ram myself unless you're just testing things out.

**Comment by wiegelmann** [comment]
Elastic won't start, and depending on your storage you need to delete the cache daily. A single-user instance can load 10 GB files per day. If you have more users, the background jobs will delay.
